app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -158,12 +158,19 @@ register_katello_host(){
 subscription-manager register <%= '--force' if truthy?(@force) %> \
   --org='<%= @organization.label if @organization %>' \
   --activationkey='<%= activation_keys %>' || <%= truthy?(@ignore_subman_errors) ? 'true' : 'cleanup_and_exit 1' %>
-
-register_katello_host | bash
-<% else -%>
-register_host | bash
 <% end -%>
 
+<%= plugin_present?('katello') ? "register_katello_host" : "register_host" %>
+reg_status=$?
```

Comment:
this is not the status if `sub-man register` (as the name implies), it's the status of `curl … -o /root/registration_host_init.sh`, is that intended?